### DISCUSSION TITLE
Is it possible to use SelectiveBloomEffect and a HDRJPG Background/Environment and have the background be masked using ignoreBackground or is a different method required?

### DISCUSSION THREAD

**Discussion by kitaedesigns** [pmndrs/postprocessing#647] [Q&A]
### Description of the bug

Trying to use SelectiveBloomEffect set at a Threshold of 2 to mask out Luminance of HDRJPG background so only the gltf asset with KHR_Emissive_strength set to 10 will be affected, but currently the background is being affected. I have tried setting up ignoreBackground in RenderPass and setting it to true in my SelectiveBloomEffect but it still is blowing out the background. It seems like Bloom is using the Luminance of the HDRJPG background? I would expect if the Threshold is set to 2 then the background would be masked from Bloom.

### To reproduce

Use an HDRJPG in scene.Background and scene.Environment and add SelectiveBloomEffect with Threshold 2, Intensity 1, Radius .5, Smoothing .025, and BlendMode set to Screen or Add. 

### Expected behavior

Expected the background to be masked from Bloom. Like this image but only the teal sphere glowing.
![image](https://github.com/user-attachments/assets/55100686-09d8-46b3-b3a0-bb15e02e5e95)

### Screenshots

![image](https://github.com/user-attachments/assets/a456d15f-c8fb-4ca8-9569-f985b7951258)

### Library versions used

 - Three: r166
 - Post Processing: 6.35.5

### Desktop
Windows, Chrome, 127.0.6533.88/89

**Comment by vanruesc** [comment]
> I would expect if the Threshold is set to 2 then the background would be masked from Bloom.

That only works if the luminance of the HDR image never exceeds 2. Judging by the results however, the image seems to contain values that are much greater than that.

> I have tried setting up ignoreBackground in RenderPass and setting it to true in my SelectiveBloomEffect but it still is blowing out the background.

`SelectiveBloomEffect` should mask out the background if `SelectiveBloomEffect.ignoreBackground` is set to `true`. `RenderPass.ignoreBackground` skips rendering the background altogether on a scene rendering level.

I'm moving this to the forum since it's a general question.
